.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: "EBMBot CI"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  unit_test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install dependencies
+        run: pip install -r requirements.txt
+      - name: "Unit tests"
+        run: pytest -v
+      - name: "Coverage"
+        run: pytest --cov=. --cov-fail-under=100
+
+  linting:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: "Linting"
+        run: |
+          pip install black==20.8b1
+          pip install flake8==3.8.3
+          pip install isort==5.5.4
```

Comment:
Any reason not to install from requirements.txt?  Feels like a potential source of confusion if requirements.txt is updated to a linter with new rules.

Reply:
Yeah, don't see why not.